pkg/kafka/kafka-publisher.go

```diff
@@ -144,10 +142,10 @@ func (p *publisher) Stop() {
 }
 
 // NewKafkaPublisher instantiates a new instance of a Kafka publisher
-func NewKafkaPublisher(kafkaSrv string) (pub.Publisher, error) {
+func NewKafkaPublisher(kConfig *Config) (pub.Publisher, error) {
```

Comment:
check for nil

Reply:
this is now checked in validator on line 147 .